src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/SuppressedIssuesFileStorageTests.cs

```diff
@@ -40,8 +40,8 @@ public class SuppressedIssuesFileStorageTests
         [TestMethod]
         public void Update_HasIssues_IssuesWrittenToFile()
         {
-            SonarQubeIssue issue1 = CreateIssue("issueKey1");
-            SonarQubeIssue issue2 = CreateIssue("issueKey2");
+            SonarQubeIssue issue1 = TestHelper.CreateIssue("issueKey1");
```

Comment:
If you make the helper class static you can add a `static using` so there is no need to include the class name. See `using static SonarQube.Client.Tests.Infra.MocksHelper;` for an example.